app/models/project.rb

```diff
@@ -200,8 +201,23 @@ def self.total
     end
   end
 
-  def test
-    puts "hello"
+  def self.fetch_lifted_projects
+    raise "Tidelift API key not set." if Rails.configuration.tidelift_api_key.blank?
+
+    current_page = 1
+    results = []
+    while !current_page.nil? && current_page < 1000 # failsafe
+      resp = Typhoeus.get("https://api.tidelift.com/external-api/packages?lifted=true&page=#{current_page}&per_page=100", headers: { "Authorization" => "Bearer #{Rails.configuration.tidelift_api_key}" })
+      break if resp.response_code != 200
```

Comment:
This break is going to cut off the results if any of the page requests fail. The rake task calling this is marking any currently lifted Projects that are missing from the list as unlifted. This break is going to cause some projects to be labeled as unlifted until the rake task runs again and is able to get through all pages.